appendix-mega65r2-flashing.tex

```diff
@@ -459,7 +459,15 @@ \section{Flashing the main FPGA using Vivado}
 
 If you choose to proceed, you will need a functioning installation of Xilinx's Vivado software, and the FTDI drivers installed, as described in the earlier sections.
 
-You will also need to download or build an .mcs bitstream file (and optional .prm checksum verification file) that you intend flash onto the QSPI chip via Vivado. See \nameref{sec:bitstreamfiles} for more details on where such files can be downloaded.
+You will also need to download or build an .mcs bitstream file (and optional
+.prm checksum verification file) that you intend flash onto the QSPI chip via
+Vivado. See
+\ifdefined\printmanual
+the {\bf MEGA65 Book}
+\else
+\bookvref{sec:bitstreamfiles}
+\fi
```

Comment:
As this `mega65r2-flashing` appendix doesn't reside within the user-guide, don't think we'll need a \printmanual check here.

As for the switch from \nameref to \bookvref, though wonder if it reads nicer with both? E.g.:

> See Bitstream files in Chapter/Appendix 5 on page 5-6 for more details...

Reply:
Please test different Latex commands yourself. You said you wanted to have your environment up and running. I expect weird things to happen if I began experiments like this. I'm not willing to test a full PR worth of changes again ATM.

I'm not changing here. You could file a new ticket.